### PR TITLE
Fix fastp merge issue with subworkflow

### PR DESCRIPTION
**Pull request by pinin4fjords**
This fixes an issue in `fastq_qc_trim_filter_setstrandedness` inherited from its origin in nf-core/rnaseq. See https://github.com/nf-core/rnaseq/issues/1452 for detail.

## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

**Review by jfy133 — DISMISSED** [review]
Assuming the save_merged schema addition is a bonus bug fix?

**Review by maxulysse — APPROVED** [review]
nice catch

File: subworkflows/nf-core/fastq_qc_trim_filter_setstrandedness/meta.yml [hunk 1]
```diff
@@ -142,6 +142,10 @@ input:
   - skip_linting:
       type: boolean
       description: Whether to skip linting of FastQ files
+  - save_merged:
```
Comment (pinin4fjords) [inline subworkflows/nf-core/fastq_qc_trim_filter_setstrandedness/meta.yml:145]:
```suggestion
  - fastp_merge:
```

File: subworkflows/nf-core/fastq_qc_trim_filter_setstrandedness/tests/main.nf.test [hunk 2]
```diff
@@ -121,6 +122,7 @@ nextflow_workflow {
                 input[20] = 0.8            // stranded_threshold
                 input[21] = 0.1            // unstranded_threshold
                 input[22] = false          // skip_linting
+                input[23] = true           // fastp_merged
```
Comment (pinin4fjords) [inline subworkflows/nf-core/fastq_qc_trim_filter_setstrandedness/tests/main.nf.test:125]:
```suggestion
                input[23] = true           // fastp_merge
```

**Comment by pinin4fjords** [comment]
> Assuming the save_merged schema addition is a bonus bug fix?

Not sure what you're getting at....

**Pull request merged**